core/src/test/scala/org/apache/spark/sql/delta/DescribeDeltaHistorySuite.scala

```diff
@@ -924,7 +924,9 @@ trait DescribeDeltaHistorySuiteBase
         val expectedMetrics = Map(
           "numConvertedFiles" -> deltaLog.snapshot.numOfFiles.toString
         )
-        val operationMetrics = getOperationMetrics(deltaTable.history(1))
+        // TODO: Remove filter once Delta API is ready with collectStats in Convert
+        val operationMetrics = getOperationMetrics(deltaTable.history()
```

Comment:
why this change. what is missing/failing without it?

Reply:
Since eventually we convert to delta with stats on a single commit, there is no need for that to make the tests pass. Removed.